web-services/client/src/main/java/datawave/webservice/query/Query.java

```diff
@@ -61,6 +61,8 @@ public abstract void initialize(String userDN, List<String> dnList, String query
     
     public abstract Set<Parameter> getParameters();
     
+    public abstract long getQueryExecutionForCurrentPageStartTime();
```

Comment:
I am wondering if we can shorten this name somehow.  Perhaps getCurrentPageStartTime ?